code/modules/atmospherics/machinery/components/gas_recipe_machines/crystallizer.dm

```diff
@@ -133,8 +132,8 @@
 ///Calculation for the heat of the various gas mixes and controls the quality of the item
 /obj/machinery/atmospherics/components/binary/crystallizer/proc/heat_calculations()
 	var/datum/gas_mixture/cooling_port = airs[1]
-	if(cooling_port.total_moles() * 0.25 > MINIMUM_MOLE_COUNT)
-		var/datum/gas_mixture/cooling_remove = cooling_port.remove_ratio(0.25)
```

Comment:
?

Reply:
i've increased the amount of cooling gas that is checked so that you can properly use the machine without too much struggle

Reply:
If you're just gonna remove 100% of the gas, why remove it at all?

Reply:
i mean is easier to handle for the sake of calculations, at least in my mind

Reply:
Are you sure about that? Try it

Reply:
should be done now :)